ci/jenkins/jobs/projects.yaml

```diff
@@ -5,6 +5,108 @@
     git_credentials_id: ANTREA_GIT_CREDENTIAL
     org_repo: antrea-io/theia
     jobs:
+      - '{name}-{test_name}-for-period':
+         test_name: daily-flow-visibility-validate
+         node: 'antrea-test-node'
+         description: 'This is for validating the flow visibility manifest daily'
+         builders:
+           - builder-e2e
+         branches:
+         - '*/main'
+         included_regions: []
+         cron: 'H H * * *'
+         ignore_post_commit_hooks: false
+         publishers:
+         - email:
+             recipients: projectantrea-dev@goolegroups.com
+         wrappers:
+         - timeout:
+             fail: true
+             timeout: 150
+             type: absolute
+         - credentials-binding:
+           - text:
+               credential-id: VMC_NETWORK_0
+               variable: VMC_NETWORK_0
+           - text:
+               credential-id: VMC_NETWORK_1
+               variable: VMC_NETWORK_1
+           - text:
+               credential-id: VCENTERNAME
+               variable: VCENTERNAME
+           - text:
+               credential-id: DATACENTERNAME
+               variable: DATACENTERNAME
+           - text:
+               credential-id: RESOURCEPOOLPATH
+               variable: RESOURCEPOOLPATH
+           - text:
+               credential-id: CAPVC_USERNAME
+               variable: CAPVC_USERNAME
+           - text:
+               credential-id: CAPVC_PASSWORD
+               variable: CAPVC_PASSWORD
+           - text:
+               credential-id: DATASTORE
+               variable: DATASTORE
+           - text:
+               credential-id: VMFOLDERNAME
+               variable: VMFOLDERNAME
+           - text:
+               credential-id: THEIA_DOCKERHUB_USERNAME
```

Comment:
Just to confirm: you have created THEIA_DOCKERHUB_USERNAME and THEIA_DOCKERHUB_TOKEN credentials on the lab Jenkins.

Reply:
yes, i used my company account for now.